### PR TITLE
Bugfix/KITOSUDV-3632

### PR DESCRIPTION
**Pull request by a-naskret**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/app/components/home/home-edit-user.view.html [hunk 1]
```diff
@@ -16,18 +16,24 @@ <h3>Profilindstillinger</h3>
...
                 <!-- User name -->
-                <div class="form-group">
+                <div class="form-group" ng-class="{ 'has-error': profileForm.name.$invalid }">
                     <label for="username" class="control-label">
                         Fornavn
                     </label>
-                    <input type="text" id="username" data-ng-model="user.name" class="form-control" data-ng-blur="patch('name', user.name)" />
+                    <input type="text" id="username" name="name" data-ng-model="user.name" data-ng-required="true" class="form-control" data-ng-blur="patch(nameFieldName, user.name)" />
+                    <div ng-messages="profileForm.name.$error" ng-if="profileForm.name.$invalid" class="color-maroon" role="alert">
+                        <div ng-message="required" style="margin:5px 0">Name is required.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/home/home-edit-user.view.html:25]: text to danish

File: Presentation.Web/app/components/home/home-edit-user.view.html [hunk 2]
```diff
@@ -16,18 +16,24 @@ <h3>Profilindstillinger</h3>
...
                 <!-- User lastname -->
-                <div class="form-group">
+                <div class="form-group" ng-class="{ 'has-error': profileForm.lastName.$invalid }">
                     <label for="lastname" class="control-label">
                         Efternavn
                     </label>
-                    <input type="text" id="lastname" data-ng-model="user.lastName" class="form-control" data-ng-blur="patch('lastName', user.lastName)" />
+                    <input type="text" id="lastname" name="lastName" data-ng-model="user.lastName" data-ng-required="true" class="form-control" data-ng-blur="patch(lastNameFieldName, user.lastName)" />
+                    <div ng-messages="profileForm.lastName.$error" ng-if="profileForm.lastName.$invalid" class="color-maroon" role="alert">
+                        <div ng-message="required" style="margin:5px 0">Last name is required.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/home/home-edit-user.view.html:35]: text to danish

File: Presentation.Web/app/components/home/home-edit-user.view.html [hunk 3]
```diff
@@ -37,11 +43,15 @@ <h3>Profilindstillinger</h3>
...
-                <div class="form-group">
+                <div class="form-group" ng-class="{ 'has-error': profileForm.email.$invalid }">
                     <label for="usermail" class="control-label">
                         Email
                     </label>
-                    <input type="text" id="usermail" data-ng-model="user.email" class="form-control" data-ng-blur="patch('email', user.email)" />
+                    <input type="text" id="usermail" name="email" data-ng-model="user.email" data-ng-required="true" class="form-control" data-ng-blur="patch(emailFieldName, user.email)" />
+                    <div ng-messages="profileForm.email.$error" ng-if="profileForm.email.$invalid" class="color-maroon" role="alert">
+                        <div ng-message="required" style="margin:5px 0">Email is required.</div>
+                        <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/home/home-edit-user.view.html:53]: texts to danish

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 4]
```diff
@@ -6,37 +6,56 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
                 Email-adresse
             </label>
             <div class="col-xs-8">
-                <input type="email" name="email" id="email" data-ng-model="ctrl.vm.email" class="form-control" data-ng-required="true" data-unique-org-user data-unique-email autofocus data-ng-disabled="ctrl.busy" maxlength="100">
-                <span class="glyphicon form-control-feedback" data-ng-class="{ 'glyphicon-remove': ctrl.createForm.name.$invalid && ctrl.createForm.name.$dirty, 'glyphicon-ok': ctrl.createForm.name.$valid }"></span>
+                <input type="email" name="email" id="email" data-ng-model="ctrl.vm.email" class="form-control" ng-required="true" data-unique-org-user data-unique-email autofocus data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.email.$error" ng-if="createUser.form.email.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Email is required.</div>
+                    <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:18]: texts to danish + check if length one is correct

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 5]
```diff
@@ -6,37 +6,56 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
                 Gentag email-adresse
             </label>
             <div class="col-xs-8">
                 <input type="email" name="repeatEmail" id="repeatEmail" data-ng-model="ctrl.repeatEmail" class="form-control" data-match="ctrl.vm.email" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div>{{createUser.form.repeatEmail.$error}}</div>
+                <div ng-messages="createUser.form.repeatEmail.$error" ng-if="createUser.form.repeatEmail.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Repeat email is required.</div>
+                    <div ng-message="unique" style="margin:5px 0">Emails differ.</div>
+                    <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:33]: texts to danish + check if length one is correct

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 6]
```diff
@@ -6,37 +6,56 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
-        <div class="form-group" data-ng-hide="emailExists">
+        <div class="form-group" ng-class="{ 'has-error': createUser.form.name.$invalid }" data-ng-hide="emailExists">
             <label for="name" class="col-xs-3 control-label">
                 Fornavn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="name" id="name" data-ng-model="ctrl.vm.name" class="form-control" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.name.$error" ng-if="createUser.form.name.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Name is required.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:45]: texts to danish + check if length one is correct

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 7]
```diff
@@ -6,37 +6,56 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
-        <div class="form-group" data-ng-hide="emailExists">
+        <div class="form-group"  ng-class="{ 'has-error': createUser.form.name.$invalid }" data-ng-hide="emailExists">
             <label for="lastName" class="col-xs-3 control-label">
                 Efternavn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="lastName" id="lastName" data-ng-model="ctrl.vm.lastName" class="form-control" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.lastName.$error" ng-if="createUser.form.lastName.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Last name is required.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:57]: texts to danish + check if length one is correct

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 8]
```diff
@@ -6,28 +6,41 @@ <h3 class="pull-left">Redigér bruger</h3>
...
+        <div class="form-group" ng-class="{ 'has-error': editUser.form.email.$invalid }">
             <label for="email" class="col-xs-3 control-label">
                 Email-adresse
             </label>
             <div class="col-xs-8">
                 <input type="email" name="email" id="email" data-ng-model="ctrl.vm.email" class="form-control" data-ng-required="true" data-disabled-on-busy maxlength="100">
+                <div ng-messages="editUser.form.email.$error" ng-if="editUser.form.email.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Email is required.</div>
+                    <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:18]: texts to danish + check if length one is correct

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 9]
```diff
@@ -6,28 +6,41 @@ <h3 class="pull-left">Redigér bruger</h3>
...
-        <div class="form-group">
+        <div class="form-group" ng-class="{ 'has-error': editUser.form.name.$invalid }">
             <label for="name" class="col-xs-3 control-label">
                 Navn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="name" id="name" data-ng-model="ctrl.vm.name" class="form-control" data-ng-required="true" data-disabled-on-busy maxlength="100">
+                <div ng-messages="editUser.form.name.$error" ng-if="editUser.form.name.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Name is required.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:30]: texts to danish + check if length one is correct

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 10]
```diff
@@ -6,28 +6,41 @@ <h3 class="pull-left">Redigér bruger</h3>
...
-        <div class="form-group">
+        <div class="form-group" ng-class="{ 'has-error': editUser.form.lastName.$invalid }">
             <label for="lastName" class="col-xs-3 control-label">
                 Efternavn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="lastName" id="lastName" data-ng-model="ctrl.vm.lastName" class="form-control" data-ng-required="true" data-disabled-on-busy maxlength="100">
+                <div ng-messages="editUser.form.lastName.$error" ng-if="editUser.form.lastName.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Last name is required.</div>
+                    <div ng-message="maxlength" style="margin:5px 0">Det valgte navn er for lang.</div>
```
Comment (a-naskret) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:42]: texts to danish + check if length one is correct

File: Presentation.Web/Content/less/kitos.less [hunk 11]
```diff
@@ -1661,4 +1661,8 @@ tbody.bordered > tr > td {
 
 .wide-modal .modal-dialog {
     width: 1000px !important;
+}
+
+.color-maroon{
```
Comment (mrjsawdk) [inline Presentation.Web/Content/less/kitos.less:1666]: perhaps introduce a term in sead... perhaps: input-validation-error or something like that

File: Presentation.Web/app/components/home/home-edit-user.controller.ts [hunk 12]
```diff
@@ -65,6 +69,18 @@
...
             $scope.patch = function (field, value) {
+
+                if (value === undefined) {
+                    if (field === $scope.nameFieldName ||
+                        field === $scope.lastNameFieldName ||
+                        field === $scope.emailFieldName) {
+                        return;
+                    }
+                }
+                if (field === 'name' && value === undefined) {
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/home/home-edit-user.controller.ts:80]: why not use the `$scope.nameFieldName` as in the condition above?
And why check it twice.. should already be included in the previous check right?

File: Presentation.Web/app/components/home/home-edit-user.view.html (hunk 1 above)
Comment (mrjsawdk) [reply Presentation.Web/app/components/home/home-edit-user.view.html:25]: Indtast fornavn

File: Presentation.Web/app/components/home/home-edit-user.view.html (hunk 2 above)
Comment (mrjsawdk) [reply Presentation.Web/app/components/home/home-edit-user.view.html:35]: Indtast efternavn

File: Presentation.Web/app/components/home/home-edit-user.view.html (hunk 3 above)
Comment (mrjsawdk) [reply Presentation.Web/app/components/home/home-edit-user.view.html:53]: Den indtastede email er ugyldig

File: Presentation.Web/app/components/home/home-edit-user.view.html [hunk 13]
```diff
@@ -37,11 +43,15 @@ <h3>Profilindstillinger</h3>
...
                 <!-- User email -->
-                <div class="form-group">
+                <div class="form-group" ng-class="{ 'has-error': profileForm.email.$invalid }">
                     <label for="usermail" class="control-label">
                         Email
                     </label>
-                    <input type="text" id="usermail" data-ng-model="user.email" class="form-control" data-ng-blur="patch('email', user.email)" />
+                    <input type="text" id="usermail" name="email" data-ng-model="user.email" data-ng-required="true" class="form-control" data-ng-blur="patch(emailFieldName, user.email)" />
+                    <div ng-messages="profileForm.email.$error" ng-if="profileForm.email.$invalid" class="color-maroon" role="alert">
+                        <div ng-message="required" style="margin:5px 0">Email is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/home/home-edit-user.view.html:52]: Email er påkrævet

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 14]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
+        <div class="form-group" ng-class="{ 'has-error': createUser.form.email.$invalid }">
             <label for="email" class="col-xs-3 control-label">
                 Email-adresse
             </label>
             <div class="col-xs-8">
-                <input type="email" name="email" id="email" data-ng-model="ctrl.vm.email" class="form-control" data-ng-required="true" data-unique-org-user data-unique-email autofocus data-ng-disabled="ctrl.busy" maxlength="100">
-                <span class="glyphicon form-control-feedback" data-ng-class="{ 'glyphicon-remove': ctrl.createForm.name.$invalid && ctrl.createForm.name.$dirty, 'glyphicon-ok': ctrl.createForm.name.$valid }"></span>
+                <input type="email" name="email" id="email" data-ng-model="ctrl.vm.email" class="form-control" ng-required="true" data-unique-org-user data-unique-email autofocus data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.email.$error" ng-if="createUser.form.email.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Email is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:16]: email er påkrævet

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 15]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
+                    <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:17]: Den indtastede email er ugyldig

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 16]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
+        <div class="form-group" ng-class="{ 'has-error': createUser.form.repeatEmail.$invalid }" data-ng-hide="emailExists">
             <label for="repeatEmail" class="col-xs-3 control-label">
                 Gentag email-adresse
             </label>
             <div class="col-xs-8">
                 <input type="email" name="repeatEmail" id="repeatEmail" data-ng-model="ctrl.repeatEmail" class="form-control" data-match="ctrl.vm.email" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.repeatEmail.$error" ng-if="createUser.form.repeatEmail.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Repeat email is required.</div>
+                    <div ng-message="unique" style="margin:5px 0">Emails differ.</div>
+                    <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:30]: Den indtastede email er ugyldig

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 17]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
         </div>
-        <div class="form-group" data-ng-hide="emailExists">
+        <div class="form-group" ng-class="{ 'has-error': createUser.form.repeatEmail.$invalid }" data-ng-hide="emailExists">
             <label for="repeatEmail" class="col-xs-3 control-label">
                 Gentag email-adresse
             </label>
             <div class="col-xs-8">
                 <input type="email" name="repeatEmail" id="repeatEmail" data-ng-model="ctrl.repeatEmail" class="form-control" data-match="ctrl.vm.email" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.repeatEmail.$error" ng-if="createUser.form.repeatEmail.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Repeat email is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:28]: Gentag den indtastede email

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 18]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
+                    <div ng-message="unique" style="margin:5px 0">Emails differ.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:29]: De to emails er forskellige

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 19]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
         </div>
-        <div class="form-group" data-ng-hide="emailExists">
+        <div class="form-group" ng-class="{ 'has-error': createUser.form.name.$invalid }" data-ng-hide="emailExists">
             <label for="name" class="col-xs-3 control-label">
                 Fornavn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="name" id="name" data-ng-model="ctrl.vm.name" class="form-control" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.name.$error" ng-if="createUser.form.name.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Name is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:41]: Indtast fornavn

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 20]
```diff
@@ -6,28 +6,38 @@ <h3 class="pull-left">Redigér bruger</h3>
...
     <form name="editUser.form" novalidate="novalidate" class="form-horizontal">
-        <div class="form-group">
+        <div class="form-group" ng-class="{ 'has-error': editUser.form.email.$invalid }">
             <label for="email" class="col-xs-3 control-label">
                 Email-adresse
             </label>
             <div class="col-xs-8">
                 <input type="email" name="email" id="email" data-ng-model="ctrl.vm.email" class="form-control" data-ng-required="true" data-disabled-on-busy maxlength="100">
+                <div ng-messages="editUser.form.email.$error" ng-if="editUser.form.email.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Email is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:16]: Email skal indtastes

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 21]
```diff
@@ -6,28 +6,38 @@ <h3 class="pull-left">Redigér bruger</h3>
...
+                    <div ng-message="email" style="margin:5px 0">Email is incorrect.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:17]: Den indtastede email er ugyldig

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 22]
```diff
@@ -6,28 +6,38 @@ <h3 class="pull-left">Redigér bruger</h3>
...
         </div>
-        <div class="form-group">
+        <div class="form-group" ng-class="{ 'has-error': editUser.form.name.$invalid }">
             <label for="name" class="col-xs-3 control-label">
                 Navn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="name" id="name" data-ng-model="ctrl.vm.name" class="form-control" data-ng-required="true" data-disabled-on-busy maxlength="100">
+                <div ng-messages="editUser.form.name.$error" ng-if="editUser.form.name.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Name is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:28]: Indtast fornavn

File: Presentation.Web/app/components/org/user/org-user-edit.modal.view.html [hunk 23]
```diff
@@ -6,28 +6,38 @@ <h3 class="pull-left">Redigér bruger</h3>
...
         </div>
-        <div class="form-group">
+        <div class="form-group" ng-class="{ 'has-error': editUser.form.lastName.$invalid }">
             <label for="lastName" class="col-xs-3 control-label">
                 Efternavn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="lastName" id="lastName" data-ng-model="ctrl.vm.lastName" class="form-control" data-ng-required="true" data-disabled-on-busy maxlength="100">
+                <div ng-messages="editUser.form.lastName.$error" ng-if="editUser.form.lastName.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Last name is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-edit.modal.view.html:39]: Indtast efternavn

File: Presentation.Web/app/components/org/user/org-user-create.modal.view.html [hunk 24]
```diff
@@ -6,37 +6,51 @@ <h3 class="modal-title pull-left">Opret bruger</h3>
...
         </div>
-        <div class="form-group" data-ng-hide="emailExists">
+        <div class="form-group"  ng-class="{ 'has-error': createUser.form.name.$invalid }" data-ng-hide="emailExists">
             <label for="lastName" class="col-xs-3 control-label">
                 Efternavn
             </label>
             <div class="col-xs-8">
                 <input type="text" name="lastName" id="lastName" data-ng-model="ctrl.vm.lastName" class="form-control" data-ng-required="true" data-ng-disabled="ctrl.busy" maxlength="100">
+                <div ng-messages="createUser.form.lastName.$error" ng-if="createUser.form.lastName.$invalid" class="color-maroon" role="alert">
+                    <div ng-message="required" style="margin:5px 0">Last name is required.</div>
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user-create.modal.view.html:52]: Indtast efternavn

**Review by mrjsawdk — APPROVED** [review]
@a-naskret ready once the danish texts have been added

**Pull request merged**